src/PageDocumentation/index.tsx

```diff
@@ -203,7 +203,7 @@ function ArtifactSetKeyPane() {
 }
 function CharacterKeyPane() {
   const { t } = useTranslation([...new Set(allCharacterKeys)].map(k => `char_${k}_gen`))
-  const charKeysCode = `type CharacterKey\n  = ${[...new Set(allCharacterKeys)].sort().map(k => `"${k}" //${t(`char_${k}_gen:name`)}`).join(`\n  | `)}`
+  const charKeysCode = `type CharacterKey\n  = ${[...new Set(locationCharacterKeys)].sort().map(k => `"${k}" //${k === "Traveler" ? "Traveler" : t(`char_${k}_gen:name`)}`).join(`\n  | `)}`
```

Comment:
Also, can use Traveler's translated name? Just have it format as something like
`// Aether/Lumine`